backend/pkg/core/lobby.go

```diff
@@ -8,11 +8,14 @@ import (
 type (
 	Lobby struct {
```

Comment:
You changed the Lobby struct definition but the `Create` method still accepts `restaurantId` as a parameter. Shouldn't it be pointer to Restaurant object?